<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifyPage</class>
 <widget class="QWidget" name="NotifyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>512</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabFeed">
      <attribute name="title">
       <string>Feed</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QGroupBox" name="notify_ForumNewMsg">
         <property name="title">
          <string>News Feed</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="notify_Peers">
              <property name="text">
               <string>Friend Connect</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Channels">
              <property name="text">
               <string>Channels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Forums">
              <property name="text">
               <string>Forums</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Posted">
              <property name="text">
               <string>Posted</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Blogs">
              <property name="text">
               <string>Blogs</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Messages">
              <property name="text">
               <string>Messages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Chat">
              <property name="text">
               <string>Chat</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_Security">
              <property name="text">
               <string>Security</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notify_SecurityIp">
              <property name="text">
               <string>Ip security</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="feedLayout"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="testFeedButton">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="messageGroupBox">
         <property name="title">
          <string>Message</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="message_ConnectAttempt">
            <property name="text">
             <string>Connect attempt</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabToasters">
      <attribute name="title">
       <string>Toasters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QPushButton" name="pushButtonDisableAll">
         <property name="toolTip">
          <string>Disable All Toaster temporarily</string>
         </property>
         <property name="text">
          <string>Disable All Toasters</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Toasters</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QCheckBox" name="popup_Connect">
                <property name="text">
                 <string>Friend Connect</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_NewMsg">
                <property name="text">
                 <string>New Message</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_DownloadFinished">
                <property name="text">
                 <string>Download completed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_PrivateChat">
                <property name="text">
                 <string>Private Chat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_GroupChat">
                <property name="text">
                 <string>Group Chat</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_ChatLobby">
                <property name="text">
                 <string>Chat Lobby</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="popup_ConnectAttempt">
                <property name="text">
                 <string>Connect attempt</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="toasterLayout"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QPushButton" name="testToasterButton">
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="labelToasterPosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="3">
             <widget class="QComboBox" name="comboBoxToasterPosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelToasterXMargin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>X Margin</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBoxToasterXMargin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="labelToasterYMargin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Y Margin</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QSpinBox" name="spinBoxToasterYMargin">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSystray">
      <attribute name="title">
       <string>Systray</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QGroupBox" name="trayNotifyGroupBox">
         <property name="title">
          <string>Systray Icon</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QGridLayout" name="notifyLayout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Systray message</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="systray_GroupChat">
            <property name="text">
             <string>Group chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="systray_ChatLobby">
            <property name="text">
             <string>Chat lobbies</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabChatLobbies">
      <attribute name="title">
       <string>Chat Lobbies</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QCheckBox" name="chatLobbies_CountUnRead">
         <property name="text">
          <string>Count all unread messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chatLobbies_CheckNickName">
         <property name="text">
          <string>Count occurences of my current identity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chatLobbies_CountFollowingText">
         <property name="text">
          <string>Count occurences of any of the following texts (separate by newlines):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="chatLobbies_TextToNotify"/>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>notify_Peers</tabstop>
  <tabstop>notify_Channels</tabstop>
  <tabstop>notify_Forums</tabstop>
  <tabstop>notify_Posted</tabstop>
  <tabstop>notify_Blogs</tabstop>
  <tabstop>notify_Messages</tabstop>
  <tabstop>notify_Chat</tabstop>
  <tabstop>notify_Security</tabstop>
  <tabstop>notify_SecurityIp</tabstop>
  <tabstop>message_ConnectAttempt</tabstop>
  <tabstop>testFeedButton</tabstop>
  <tabstop>popup_Connect</tabstop>
  <tabstop>popup_NewMsg</tabstop>
  <tabstop>popup_DownloadFinished</tabstop>
  <tabstop>popup_PrivateChat</tabstop>
  <tabstop>popup_GroupChat</tabstop>
  <tabstop>popup_ChatLobby</tabstop>
  <tabstop>popup_ConnectAttempt</tabstop>
  <tabstop>testToasterButton</tabstop>
  <tabstop>comboBoxToasterPosition</tabstop>
  <tabstop>spinBoxToasterXMargin</tabstop>
  <tabstop>spinBoxToasterYMargin</tabstop>
  <tabstop>systray_GroupChat</tabstop>
  <tabstop>systray_ChatLobby</tabstop>
  <tabstop>chatLobbies_CountUnRead</tabstop>
  <tabstop>chatLobbies_CheckNickName</tabstop>
  <tabstop>chatLobbies_CountFollowingText</tabstop>
  <tabstop>chatLobbies_TextToNotify</tabstop>
  <tabstop>pushButtonDisableAll</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
